doc/rtt.js

```diff
@@ -0,0 +1,178 @@
+ *    <tt>nrfjprogjs.RTTCouldNotLoadnRFjprogLibrary</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotOpennRFjprogLibrary</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotConnectToDevice</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotStartRTT</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotFindControlBlock</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotGetChannelInformation</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotCallFunction</tt><br/>
+ *    <tt>nrfjprogjs.RTTNotInitialized</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotExecuteDueToLoad</tt><br/>
+ *
+ * @property {String} errcode A human-readable version of the error code.
+ * @property {String} erroroperation The internal function that caused the error.
+ * @property {String} errmsg Error string. The value will be equal to that of the built-in <tt>message</tt> property.
+ * @property {integer} lowlevelErrorNo The low-level error code, if applicable.
+ * @property {String} lowlevelError A human-readable version of the low-level error code.
+ * @property {String} log The complete log from the internal functions.
+ *
+ * @example
+ * RTT0.start(12345678, {}, function(err, down, up){
+ *     if (err) {
+ *         throw err;
+ *     } else {
+ *         // success
+ *     }
+ * });
+ *
+ * @example
+ * RTT.start(12345678, {}, function(err, down, up){
+ *     if (err && err.errno === nrfjprogjs.RTTCouldNotFindControlBlock) {
+ *          console.error('The firmware is not RTT capable');
+ *     }
+ * });
+ */
+
+/**
+ * Information about the different up and down channels available on the device
+ * @typedef ChannelInfo
+ * @property {integer} channelIndex The index used to address this channel
+ * @property {String} name The name of the channel
+ * @property {integer} size The size of the channel
+ */
+
+ /**
+ * Option flags to be used when starting RTT. This may speed up the process of locating the control block, and the RTT Start.
+ * @typedef StartOptions
+ * @property {integer} controlBlockLocation The location of the control block. If this location is not the start of the RTT control block, start will fail.
+ */
+
+
+/**
+ * Async function to start RTT
+ *
+ * @example
+ * RTT.start(12345678, {}, function(err, down, up){
+ *      if (err) console.error('The firmware is not RTT capable');
+ *
+ *      console.log('Down channels');
+ *      down.map(function(channel) {
+ *             console.log('Index:', channel.channelIndex);
+ *             console.log('Name:', channel.name);
+ *             console.log('Size:', channel.size);
+ *      });
+ *
+ *      console.log('Up channels');
+ *      up.map(function(channel) {
+ *             console.log('Index:', channel.channelIndex);
+ *             console.log('Name:', channel.name);
+ *             console.log('Size:', channel.size);
+ *      });
+ * });
+ *
+ * @param {integer} serialNumber The serial number of the device to start RTT on
+ * @param {integer} startOptions A plain object containing options about how to start RTT
```

Comment:
I'm pretty sure this is not an `integer`

Reply:
That is correct. Should be `StartOption`. Will fix.